documents/motions/10.2021.10 - Results Team.md

```diff
@@ -0,0 +1,37 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 10.2021.10  
+**Subject:** WCA Results Team  
+**Intent:** Rights and duties of the WCA Results Team  
+**Submitted by:** WCA Board 
+**Date:** February 28, 2021  
+
+# Motion
+
+The WCA Results Team is an Advisory Committee of the WCA with the following rights and duties:
+
+1. Management of Data
+   1. Managing the data in the databases of the WCA.
+   2. Managing the administration of Registered Speedcubers and Registered Speedcuber profile pages.
+   3. Managing the data quality of the databases of the WCA.
+   4. Analysing the data in the databases of the WCA for informational and promotional purposes, and for other purposes upon request of Staff Members.
+2. Management of Results
+   1. Managing competition results and results files.
+   2. Managing corrections to competition results.
+   3. Supporting WCA Delegates to deliver high quality results files.
+3. Management of Rankings
+   1. Recognizing and administering the personal, national, continental, and world records per event.
+   2. Recognizing and administering the national, continental, and world rankings per event.
+4. All matters relating to Data Protection:
+   1. Creation, implementation, and enforcement of all WCA Data Protection policies:
+      1. Researching regulations globally to ensure the WCA adopts the policy requirements to satisfy all global regulations.
+      2. Engaging with legal council on all matters relating to Data Protection with WCA Board approval.
+      3.  Understanding the scope of which Data Protection policies are required and duly advising the WCA Board.
+   2. Acting as WCA liaison for Data Protection matters:
+      1. Managing requests from Registered Speedcubers and WCA Staff on Data Protection
+         1. Managing requests relating to data subject’s right to be forgotten.
+            1. Liaison with the WRT to request data subject’s personal data be removed or anonymized within the WCA Database. 
```

Comment:
Moving Comment on closed PR to here:
This bullet should be removed or modified as the WRT would not need to initiate communication with itself in order to carry out such a request.

Reply:
Thanks, I have removed this point.